cpp/src/merge/merge.cu

```diff
@@ -145,6 +142,16 @@ void materialize_bitmask(column_view const& left_col,
   CHECK_CUDA(stream.value());
 }
 
+void materialize_bitmask(column_view const& left_col,
```

Comment:
May be it is better to remove this, and use the new variant in all other places as well.

Reply:
Done.